src/System.Management.Automation/engine/CommandProcessorBase.cs

```diff
@@ -318,6 +318,55 @@ protected virtual void OnSetCurrentScope()
         protected virtual void OnRestorePreviousScope()
         {
         }
+        
+#if !CORECLR
```

Comment:
This project is now coreCLR only.  Why add code for FullCLR?

Reply:
@TravisEz13  See referenced issue #7408, let's continue the discussion in this issue
In order to get SMA to compile for `net472` again with more ease, those classed would help (otherwise I would need to comment out fullclr specific code).